ui/src/wizard/AzureWizardUserInput.ts

```diff
@@ -37,10 +42,10 @@ export class AzureWizardUserInput implements IWizardUserInput {
                 quickPick.step = this._wizard.currentStep;
                 quickPick.totalSteps = this._wizard.totalSteps;
             }
-            quickPick.buttons = this._wizard.currentStep > 1 ? [QuickInputButtons.Back] : [];
+            quickPick.buttons = this.showBackButton ? [QuickInputButtons.Back] : [];
 
             // Copy settings that are common between options and quickPick
-            quickPick.placeholder = options.placeHolder;
+            quickPick.placeholder = options.loadingPlaceHolder || options.placeHolder;
```

Comment:
Should we default this placeHolder to something like "Loading..." or "Retrieving..."? Or is the concern that for some quickPicks, it'd flash by so quickly that it'd be distracting?

Reply:
No I didn't have that concern - it was more that the existing placeholder already makes sense in many cases. For example, if the placeholder says "Select a Web App..." and the thing is loading - I can reasonably infer that it's loading web apps. If it just says a generic "Loading..." - I almost feel like that's less information

The ideal message is probably something like "Retrieving web apps...", but that obviously can't be set in a generic way here.